closed/OpenJ9.gmk

```diff
@@ -45,6 +45,8 @@ OPENJ9_PROPERTY_FILES := $(notdir $(wildcard $(OUTPUT_ROOT)/vm/java*.properties)
 OPENJ9_MISC_FILES := \
```

Comment:
How about we call this OPENJ9_LIB_FILES and the next one OPENJ9_VM_FILES, so the names more accurately reflect where the files are going to end up.

Reply:
@pshipton - I have renamed the variables as suggested.

```
openj9@ub14hcxvm-4:/home/openj9/openj9-openjdk-jdk8$ build/linux-x86_64-normal-server-release/images/j2sdk-image/bin/java -version
openjdk version "1.8.0-internal"
OpenJDK Runtime Environment (build 1.8.0-internal-openj9_2017_11_28_09_49-b00)
Eclipse OpenJ9 VM (build 2.9, JRE 1.8.0 Linux amd64-64 Compressed References 20171128_000000 (JIT enabled, AOT enabled)
OpenJ9   - ea57db3
OMR      - 92e9349
OpenJDK  - 7c9d7d3 based on jdk8u152-b16)
```